$NetBSD: patch-configure.ac,v 1.1 2015/05/08 12:43:31 jonthn Exp $

 * Change default exe path
 * Add various options to disable linux specific tools and ultimately only
    build 'watch'

--- configure.ac.orig	2014-09-23 11:40:36.000000000 +0000
+++ configure.ac
@@ -70,7 +70,7 @@ AC_FUNC_MMAP
 AC_FUNC_REALLOC
 AC_FUNC_STRTOD
 
-usrbin_execdir='${exec_prefix}/usr/bin'
+usrbin_execdir='${exec_prefix}/bin'
 AC_SUBST([usrbin_execdir])
 
 AM_GNU_GETTEXT_VERSION([0.14.1])
@@ -174,6 +174,12 @@ AC_ARG_ENABLE([pidof],
 )
 AM_CONDITIONAL(BUILD_PIDOF, test "x$enable_pidof" = xyes)
 
+AC_ARG_ENABLE([linux-systemtools],
+  AS_HELP_STRING([--disable-linux-systemtools], [build linux tools (top, free, ...)]),
+  [], [enable_linuxsystemtools=yes]
+)
+AM_CONDITIONAL(BUILD_LINUX_SYSTEMTOOLS, test "x$enable_linuxsystemtools" = xyes)
+
 AC_ARG_ENABLE([kill],
   AS_HELP_STRING([--disable-kill], [do not build kill]),
   [], [enable_kill=yes]
@@ -187,6 +193,24 @@ AC_ARG_ENABLE([skill],
 )
 AM_CONDITIONAL(BUILD_SKILL, test "x$enable_skill" = xyes)
 
+AC_ARG_ENABLE([slabtop],
+  AS_HELP_STRING([--enable-slabtop], [build slabtop]),
+  [], [enable_slabtop=yes]
+)
+AM_CONDITIONAL(BUILD_SLABTOP, test "x$enable_slabtop" = xyes)
+
+AC_ARG_ENABLE([watch],
+  AS_HELP_STRING([--enable-watch], [build watch]),
+  [], [enable_watch=yes]
+)
+AM_CONDITIONAL(BUILD_WATCH, test "x$enable_watch" = xyes)
+
+AC_ARG_ENABLE([test_programs],
+  AS_HELP_STRING([--enable-test_programs], [build test_programs]),
+  [], [enable_test_programs=no]
+)
+AM_CONDITIONAL(BUILD_TEST_PROGRAMS, test "x$enable_test_programs" = xyes)
+
 AC_ARG_ENABLE([examples],
   AS_HELP_STRING([--enable-examples], [add example files to installation]),
   [], [enable_examples=no]
